playbooks/roles/integreatly/defaults/main.yml

```diff
@@ -126,4 +126,13 @@ integreatly_osd_install_name: "Integreatly_Install_Template_[OSD]"
 integreatly_osd_bootstrap_name: "Integreatly_Bootstrap_Template_[OSD]"
 integreatly_osd_workflow_name: "Integreatly_Bootstrap_and_Install_[OSD]"
 integreatly_osd_workflow_description: "Workflow to configure and install integreatly on an OSD cluster"
-integreatly_osd_source_project_update_on_launch: true
\ No newline at end of file
+integreatly_osd_source_project_update_on_launch: true
+
+
+integreatly_osd_upgrade_bootstrap_name: "Integreatly_Upgrade_Bootstrap_Template_[OSD]"
+integreatly_osd_upgrade_source_name: update-project-source
+integreatly_osd_upgrade_name: "Integreatly_upgrade_[OSD]"
+integreatly_osd_upgrade_workflow_name: "Integreatly_bootstrap_and_upgrade_[OSD]"
```

Comment:
Would be worth renaming this to 'Integreatly_Bootstrap_and_Upgrade_[OSD]' to map to the other jobs